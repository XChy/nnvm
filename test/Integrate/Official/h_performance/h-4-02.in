version https://git-lfs.github.com/spec/v1
oid sha256:ca51ee7f12f9c85a76b50841cf89723ea1ec094ca8bd0f0e499e35bba5ee9d72
size 125
