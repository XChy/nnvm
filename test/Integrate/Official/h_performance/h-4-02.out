1
117520861
0
601948356
766047214
57535868
0